### ISSUE TITLE
[wt] Installed share/wt/*.cmake files contain absolute paths

### ISSUE THREAD

**Issue by koprok** [Microsoft/vcpkg#5077]
For example on Linux:
1. Download and bootstrap Vcpkg inside `/home/john/vcpkg`
1. Install Wt by running `vcpkg install wt`
1. Now the file `/home/john/vcpkg/installed/x64-linux/share/wt/wt-target-http.cmake` contains the following code:
    ```
    set_target_properties(Wt::HTTP PROPERTIES
      INTERFACE_LINK_LIBRARIES "Wt::Wt;/home/john/vcpkg/installed/x64-linux/lib/libz.a;/home/john/vcpkg/installed/x64-linux/lib/libssl.a;/home/hohn/vcpkg/installed/x64-linux/lib/libcrypto.a;\$<LINK_ONLY:Boost::thread>;\$<LINK_ONLY:Boost::program_options>;\$<LINK_ONLY:Boost::system>;\$<LINK_ONLY:Boost::filesystem>;\$<LINK_ONLY:Threads::Threads>"
    )
    ```
1. This makes it impossible to export the wt package and then use it from other path.

**Comment by studiofuga** [comment]
This is a widespread problem. In my current exported package I see also qt, cgal, libcurl have the same problem, mostly in TNTERFACE_SYSTEM_* but also in other variables.
A workaround I can see is to run a python script that search for files and fixes up the paths.
It would be a nice idea to provide a similar script in the root of the package. Note that other similar projects, like Buildroot and Yocto, do the same.

**Comment by koprok** [comment]
@studiofuga, thank you for the script. I have implemented fix for wt package only. Please see #5083 if you are interested. There is also a more general fix #4782, which is still not merged in master and I have not tried it.

**Comment by studiofuga** [comment]
Oh Thanks, that seems awesome. The script is just a workaround and must be adapted to the specific conditions. For now it fixes my needs, I'll wait for the official merge then. Thank you!

**Comment by Neumann-A** [comment]
closed by #5459 ?

**Comment by vicroms** [comment]
This has been fixed by #5459.
Please reopen if you still have any issues.
